code/style.css

```diff
@@ -1,7 +1,465 @@
+- Add a header and/or a button on top of your video, just like Confetti, by using `position: absolute;` and `z-index` to get everything in the right place.
+- Read up on the [BEM](http://getbem.com/introduction/) methodology and use it for the project
+*/
+
+body,
+footer {
+  margin: 0;
+  font-family: 'Rubik', sans-serif;
+}
+
+h1,
+p {
+  margin: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 5px;
+}
+
+/* general link styling */
+
+/* unvisited link */
+a:link {
+  color: #fff;
+}
+
+/* visited link */
+a:visited {
+  color: #444;
+}
+
+/* mouse over and selected link */
+a:hover,
+a:focus {
+  color: #DB398D;
+}
+
+/* style the header */
+
+header {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  max-height: 1080px;
+  overflow-x: hidden;
+  /* overflow-x can be used to decide what to do with content that overflows at the left and right edges */
+}
+
+/* add a heading on top of the video in the header */
+.header-content {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  position: fixed;
+  background: linear-gradient(rgba(117, 19, 93, 0.73),rgba(245, 246, 252, 0.52));
+  color: #f1f1f1;
+  width: 100%;
+  height: 100%;
+  max-height: 1080px;
+  font-family: 'Monoton', cursive;
+  text-align: center;
+}
+
+.header-content h1 {
+  font-size: 4.2rem;
+  font-weight: 400;
+}
+
+/* style the video and include a fallback image if the video does not load */
+video#headerVideo {
+  position: fixed;
+  top: 0;
+  left: 0;
+  object-fit: cover;
+  max-height: 1080px;
+  background-image: url(./xavi-cabrera-kn-UmDZQDjM-unsplash-mobile.jpg);
+}
+
+/* style the page content */
+
+article {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  width: 100vw;
+  font-size: 1.4rem;
+  position: relative;
+  margin: 0;
+}
+
+.text-section p {
+  background: #f0f9fa;
+  font-size: 1.6rem;
+  text-align: center;
+  padding: 60px 60px 60px 60px;
+  margin: 0;
+}
+
+.block-section {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  background: #fff;
+  padding: 20px 20px 80px 20px;
+}
+
+.block-section h2 {
+  text-align: center;
+  font-size: 1.8rem;
+}
+
+/* accordion */
+
+/* Code adapted from Webdevtrick ( https://webdevtrick.com ) */
+
+.accordion {
+  padding-top: 40px;
+}
+
+.accordion a {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  color: #444;
+  border-bottom: 1px solid #ccc;
+  cursor: pointer;
+  padding: 1.4em;
+  text-align: left;
+  outline: none;
+}
+
+.accordion a:hover,
+.accordion a:hover::after {
+  background-color: #ccc; 
+}
+
+.accordion a.active {
+  background-color: #ccc;
```

Comment:
Nice to keep the open/active accordions background color same as while hovering.